### PR TITLE
Desktop version #1

### PR DESCRIPTION
**Pull request by ethred**
- Add index.html with in the structure listed in the required document
- Add style.css regarding the document and the html file needs
- Add readme.md with in information about the project 
- Add all required images for the pages

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -2,110 +2,566 @@ html,
...
+.flex-col {
+  display: flex;
+  background: #fff;
+  border-radius: 24px;
+  height: auto;
+  margin-left: auto;
+  margin-right: auto;
+  padding-bottom: 1rem;
+  padding-right: 1.5rem;
+}
```
Comment (AlphaNtihinduka) [inline style.css:129]: - [ ] You have done great to ensure that the contents of the card are displayed flex. However, this style should only be applied to desktop version of your portifolio. In addition, your cards are causing horizontal scroll of the portifolio. You should also make sure that the left and right covers around 47% width of your card.
- [ ] Remember, the experience of scrolling horizontally can makes the screen judder and create a headache inducing feeling 😞 
- [ ] Please ensure that the design of the card matches exactly the design of the Figma template design in both mobile and desktop versions of your portifolio ☺️.

Your design 😎 desktop
image

Mobile
image

Figma design 😎 
image

File: style.css [hunk 2]
```diff
@@ -2,110 +2,566 @@ html,
...
+  font-size: 17px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.bottom-sec .section-bottom .right {
+  width: 60%;
+  padding: 3rem auto 0 5rem;
+}
```
Comment (AlphaNtihinduka) [inline style.css:445]: - [ ] I have noticed that the languages are not responsive to different desktop screen sizes. I suggest that you consider, fixing the responsiveness such that your design matches the Figma template design ☺️.
- [ ] Please make sure that your mobile design still looks the same way it was.
Your design 😎 
image

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @ethred 🙋‍♂️
Good job working on desktop version of your portifolio. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- Used media queries ✔️ 
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
- Kindly consider the inline review comments ☺️
- As the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md), you should use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px). 
- In addition, the mobile design should still look like expected ☺️ 
- Finally as stated and explained in the inline review comments, you should make sure that your portifolio doesn't have the horizontal scroll.

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @ethred, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joel-kalema — COMMENTED** [review]
Hi @ethred 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request  🛑 

pirates-of-the-caribbean-johnny-depp


- [ ] You have submitted a project that has has conflicts that must be resolved. 

Screenshot 2023-06-15 151608


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @joel-kalema in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**